packages/core/useSSRWidth/index.ts

```diff
@@ -0,0 +1,9 @@
+let SSRWidth: number | undefined
+
+export function useSSRWidth() {
+  return SSRWidth
+}
+
+export function setSSRWidth(width: typeof SSRWidth) {
+  SSRWidth = width
```

Comment:
I am not so sure about having a global variable for this. It would be fine for an app that only has a single fixed width on SSR. But when the width could change per request, this would lead to context leak and undetermined SSR result. We could use provide/inject to store this value if we want to reuse it across multiple composables

Reply:
Indeed, I guess it would also be subject to race conditions if the app renders multiple things async at the same time

Changed it to provide inject